devices/micromatic.js

```diff
@@ -19,7 +19,6 @@ module.exports = [
             const endpoint = device.getEndpoint(1);
             await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl', 'haElectricalMeasurement', 'seMetering']);
             await reporting.brightness(endpoint);
-            await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);
```

Comment:
This could potentially screw up the reported measurements. Could you make sure it doesn't by force removing the device via the frontend, restart z2m and repair the device. Are the reported measurements correct?

Reply:
@Koenkk is there any better way to avoid the error when including the device? Open for suggestions.

Also, did you see the semetering issue?

Reply:
Did you check if the measurements are still correct after force removing, restart z2m and repair the device? If yes, it can be removed safely.

About the metering, this is because no divisor/multiplier is available, try adding `await reporting.readMeteringMultiplierDivisor(endpoint);` here.

Reply:
Seems like your right @Koenkk , it needs the multipliers to get the correct sum.
My testing has been flawed here. 

Other suggestions to avoid the error messages for users? 
it works but not ideal i think
```await reporting.readMeteringMultiplierDivisor(endpoint); 

seems to have worked for usage also

Reply:
found some examples for targeting multipliers/divisors that we need. 
Picked the units based on the interview from the logs.